library/coverage/result_spec.rb

```diff
@@ -0,0 +1,31 @@
+require File.expand_path('../../../spec_helper', __FILE__)
+config_file = File.expand_path('../fixtures/start_coverage.rb', __FILE__)
+
+describe "Coverage.result" do
+  it "needs to be reviewed for spec completeness"
+
+  it 'gives the covered files as a hash with arrays' do
+    $LOADED_FEATURES.delete(config_file)
+    require 'coverage'
+    Coverage.start
+    require config_file.chomp('.rb')
+    result = Coverage.result
+
+    result.should == { config_file => [1, 1, 1] }
+  end
+
+  it 'should list coverage for the required file starting coverage' do
```

Comment:
I think there should be a spec mentioning that coverage is only tracked for `load/require` after a Coverage.start since it seems this spec relies on that.